util/missingcrypto.txt

```diff
@@ -1327,6 +1327,7 @@ X509_policy_tree_level_count(3)
 X509_print(3)
 X509_print_ex(3)
 X509_print_ex_fp(3)
+X509_print_ex_brief(3)
```

Comment:
Isnt it kind of forbidden to add new functions here?

Reply:
Yes! Good catch!

Reply:
I was already expecting a comment here. When introducing this auxiliary function (in order to fulfill a structural change request my @mattcaswell) I was considering trying to add documentation for it, but I decided against it because this function is strongly related to the functions around it, all of which are not (yet) documented. So I see it as consequent not (yet) to document it either.

Since we likely cannot wait for the other functions to finally get documented (ideally by their author; others like me would not very be suitable fill the gap), I see two options now - which one do you prefer?
* I try to produce some basic documentation for the new function `X509_print_ex_brief()`
* we un-export the new function - in this case: in which header file could we place its declaration such that it is available to the CMP implementation?

Reply:
> we un-export the new function - in this case: in which header file could we place its declaration such that it is available to the CMP implementation?

By this I assume you mean it needs to be available to the cmp command line app? In which case it kind of needs to be in a public header.

Therefore I prefer option (1) (produce some basic documentation)

Reply:
The CLI will use it only indirectly - via the certificate verification print callback, which is has become part of libcrypto.
Nevertheless I can implement (1).

Reply:
> The CLI will use it only indirectly - via the certificate verification print callback, which is has become part of libcrypto.

Ah. In that case. I might change my mind. Do we actually need to export this? I like to keep the external API to a minimum.

Reply:
Fine. Then again my question:
in which header file could we place its declaration such that the internal function is usable for all modules of libcrypto?

Reply:
It could go in include/crypto/x509.h (Note that it should probably be in lowercase if it is an internal function)

Reply:
Done internalizing ` {X,x}509_print_ex_brief()` .